testsuite/features/profiles/Kiwi/POS_Image-JeOS6/config.xml

```diff
@@ -30,10 +30,10 @@
       <file name="drivers/block/virtio_blk.ko" />
     </drivers>
     <repository type="rpm-md" >
-        <source path="http://euklid.nue.suse.com/mirror/SuSE/build.suse.de/SUSE/Products/SLE-SERVER/12-SP3/x86_64/product/"/>
+        <source path="http://euklid.nue.suse.com/mirror/SuSE/build.suse.de/SUSE/Products/SLE-SERVER/12-SP4/x86_64/product/"/>
     </repository>
     <repository type="rpm-md" >
-        <source path="http://euklid.nue.suse.com/mirror/SuSE/build.suse.de/SUSE/Updates/SLE-SERVER/12-SP3/x86_64/update/"/>
+        <source path="http://euklid.nue.suse.com/mirror/SuSE/build.suse.de/SUSE/Updates/SLE-SERVER/12-SP4/x86_64/update/"/>
```

Comment:
You could try to use download.suse.de/ibs url instead of euklid. Download.suse.de should have the same content and is mirrored in US. It could speed-up image building when it run in the Provo Datacenter.

Reply:
Will do.